shadow-dom/Extensions-to-Event-Interface.html

```diff
@@ -54,7 +54,6 @@
 
         var expectedPath = ['A1a', 'A1-SR', 'A1', 'A-SR', 'A'];
         assert_array_equals(log.eventPath, expectedPath);
-        assert_array_equals(log.eventPath.length, log.pathAtTargets.length);
```

Comment:
We need to replace `assert_array_equals` with `assert_equals`, not remove it.
r- because we clearly need this assertion.

Reply:
The `assert_array_equals` is already asserting the length of the array. https://github.com/web-platform-tests/wpt/blob/master/resources/testharness.js#L1052-L1055

Reply:
My bad, I added the `assert_equals` back.